# HG changeset patch
# Parent  7aaac1367125b1c2e5467dc949eea88e8164387a
Bump the device timeout to 2.5x device force refresh

This will let us do more retries before we drop devices.

diff --git a/lifx/timer.h b/lifx/timer.h
--- a/lifx/timer.h
+++ b/lifx/timer.h
@@ -20,7 +20,7 @@
 enum { LGTD_LIFX_TIMER_WATCHDOG_INTERVAL_MSECS = 500 };
 enum { LGTD_LIFX_TIMER_ACTIVE_DISCOVERY_INTERVAL_MSECS = 2000 };
 enum { LGTD_LIFX_TIMER_PASSIVE_DISCOVERY_INTERVAL_MSECS = 10000 };
-enum { LGTD_LIFX_TIMER_DEVICE_TIMEOUT_MSECS = 3000 };
+enum { LGTD_LIFX_TIMER_DEVICE_TIMEOUT_MSECS = 5000 };
 enum { LGTD_LIFX_TIMER_DEVICE_FORCE_REFRESH_MSECS = 2000 };
 
 bool lgtd_lifx_timer_setup(void);
